css/structure/jquery.mobile.forms.select.css

```diff
@@ -66,6 +66,10 @@
 .ui-select .ui-btn-icon-right .ui-li-count {
 	right: 3.2em;
 }
+/*Fixes #7918 : Count indicator position must be different for mini version*/
```

Comment:
Don't reference issues in comments. Blank line before comments. Space before first word.